servers/src/main/java/tachyon/master/balancer/Balancer.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.master.balancer;
+
+import java.net.UnknownHostException;
+import java.util.Map;
+
+import com.google.common.base.Throwables;
+
+import tachyon.Constants;
+import tachyon.conf.TachyonConf;
+import tachyon.master.MasterWorkerInfo;
+import tachyon.thrift.NetAddress;
+import tachyon.util.CommonUtils;
+
+/**
+ * Interface for the balancing policy on workers for write.
```

Comment:
This is not necessarily for a write. A client connects to a worker, to do reads and writes.